contents/customers/opensauced.md

```diff
@@ -0,0 +1,73 @@
+---
+title: How OpenSauced is moving to product-market fit with PostHog for Startups
+customer: OpenSauced
+logo: ../images/customers/opensauced/logo.svg
+logoDark: ../images/customers/opensauced/logo_dark.svg
+featuredImage: ../images/customers/opensauced/featured.png
+featuredCustomer: false
+industries:
+    - SaaS, Devtool
+users:
+    - Leadership
+    - Investors
+    - Founder
+    - Leadership
+    - Marketing
+    - Design
+    - Engineering
+toolsUsed:
+    - Analytics
+    - Cohorts
+    - PostHog Cloud
+    - Insights
+date: 2023-09-03
+---
+
+When Brian Douglas left his role as GitHub’s Director of Developer Advocacy in 2022, he knew he wanted to build a platform that would help engineers find and track their open source contributions. He called it [Opensauced.pizza](https://opensauced.pizza/) and he’d been working on the idea for years in his spare time. 
+
+“I actually added PostHog to Opensauced even before we started raising money,” says Brian. “That ended up being one of the best things I’d ever done too, because I could track the weekly active users in some of the default insights. It helped me validate the idea even before we started raising and building the team.”
```

Comment:
Please reference OpenSauced with a capital S throughout.

Reply:
Just pushing that in now - wasn't sure how you wanted to brand it, but easy to add in. 👍